warehouse/internal/repo/staging.go

```diff
@@ -0,0 +1,288 @@
+package repo
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/rudderlabs/rudder-server/utils/timeutil"
+	"github.com/rudderlabs/rudder-server/warehouse/internal/model"
+	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
+)
+
+const tableName = warehouseutils.WarehouseStagingFilesTable
```

Comment:
We can't use generic constants here, right?
It will collide with other tables later right?